src/transforms/ReplaceFieldWithFragment.ts

```diff
@@ -119,3 +129,30 @@ function parseFragmentToInlineFragment(
 
   throw new Error('Could not parse fragment');
 }
+
+function concatInlineFragments(
```

Comment:
Ok, few issues here.

* If fragments have conflicting aliases this will break
* Maybe this should dedupe fields (also watch out for aliases in that case)